src/routes/(protected)/unit/[id]/+page.svelte

```diff
@@ -77,6 +84,16 @@
       player.toggle();
     }
   };
+
+  const handleSentiment = (newSentiment: boolean | null) => {
+    sentiment = newSentiment;
+  };
+
+  const updateSentiment: SubmitFunction = async () => {
```

Comment:
I think can remove the entire function. Just use `use:enhance` in the form should give you the same outcome.